host_core/native/hostcore_wasmcloud_native/Cargo.toml

```diff
@@ -10,8 +10,7 @@ path = "src/lib.rs"
 crate-type = ["dylib"]
 
 [dependencies]
-# Patched temporarily to fix TinyGo modules, branch will be removed in v0.63.0
-wasmcloud = { git = "https://github.com/wasmcloud/wasmcloud", branch = "fix/0.62-tinygo"}
+wasmcloud-host = { git = "https://github.com/rvolosatovs/wasmcloud", branch = "feat/wit"}
```

Comment:
Calling this wasmCloud host is confusing as there is a full elixir project in this repo called wasmCloud host (the washboard). 

Also, we shouldn't be depending on your fork.

Reply:
This is a draft and the fork is used precisely because https://github.com/wasmCloud/wasmCloud/pull/300 is not merged yet. There's nothing else to depend upon to get this up